Logos/OrcaSecurity.svg

```diff
@@ -0,0 +1 @@
1-11.91-11.4a12.87,12.87,0,0,1,13-13.12,11.51,11.51,0,0,1,10,5.46L225,112.16a7.59,7.59,0,0,0-6.39-3.58,8.28,8.28,0,0,0-8,8.28,6.74,6.74,0,0,0,7.1,7,8.41,8.41,0,0,0,6.59-3.27Z"/><path d="M230.78,118.6a27.43,27.43,0,0,1,.41-4.19l1.74-9.91h4.9l-1.74,9.91a25.62,25.62,0,0,0-.4,4.13,5.13,5.13,0,0,0,5.41,5.37,9.59,9.59,0,0,0,7.31-3.83l2.7-15.58H256L251.88,128h-4.65l.41-2a10.76,10.76,0,0,1-7,2.56,9.84,9.84,0,0,1-9.86-9.91Z"/><path d="M274.15,108.74a5.91,5.91,0,0,0-1.38-.16,9.55,9.55,0,0,0-7.35,4.09L262.76,128h-4.9L262,104.5h4.65l-.46,2.24a10.51,10.51,0,0,1,6.7-2.75,8.07,8.07,0,0,1,2.6.4Z"/><path d="M283.6,104.5,279.46,128h-4.9l4.14-23.5Zm-1.07-11.75A3.21,3.21,0,0,1,285.75,96a3.16,3.16,0,0,1-3.22,3.16A3.12,3.12,0,0,1,279.36,96,3.16,3.16,0,0,1,282.53,92.75Z"/><path d="M301.89,128a11.2,11.2,0,0,1-3.47.51c-4.14,0-8.38-2.56-8.38-7.56a30.21,30.21,0,0,1,.41-3.38l1.53-8.73h-6.13l.76-4.34h6.14l1.17-6.64h4.9l-1.17,6.64h7.61l-.77,4.34h-7.61l-1.63,9.45a23.18,23.18,0,0,0-.36,2.81c0,2,1.58,3.06,3.63,3.06a9.34,9.34,0,0,0,2.45-.35Z"/><path d="M331.21,104.5l-21.92,35.76h-5.56l9-14.77-5.82-21h5L316.14,120l9.5-15.48Z"/><path d="M150.37,21a30.58,30.58,0,0,0-7.9-.84c-9.57,0-17.13,6.89-19.82,11.76V83.16H100.81V1.68h21V9.91C126.68,3.86,135.58,0,145.5,0a45.15,45.15,0,0,1,7.39.67Z"/><path d="M239.75,66.53c-9.58,12.77-21.68,18.31-37,18.31-25.2,0-43.68-18.82-43.68-42.34,0-23.68,19-42.5,43.84-42.5a43.46,43.46,0,0,1,34.28,16.3L221.77,29.9c-4.37-5.71-10.92-9.74-18.82-9.74-12.6,0-22,9.91-22,22.34,0,12.77,9.24,22.18,22.51,22.18,8.9,0,16-5.38,19.49-10.75Z"/><path d="M331.8,83.16h-21V76.61a34.06,34.06,0,0,1-22.68,8.23,41,41,0,0,1-30.74-13.78A43.75,43.75,0,0,1,247,42.5c0-24,17.81-42.5,40-42.5A37.29,37.29,0,0,1,310.8,8.23V1.68h21ZM310,30.24A25,25,0,0,0,289.8,20.16c-12.76,0-21,10.58-21,22.34,0,12.77,9.24,22.18,21.51,22.18A23.85,23.85,0,0,0,310,54.6Z"/><path d="M43.34,0A45.2,45.2,0,0,0,27.52,2.81a47.9,47.9,0,0,1,14,17.43c.6,0,1.21-.08,1.82-.08,12.77,0,21.51,10.08,21.51,22.34S56.11,64.68,43.34,64.68a22.67,22.67,0,0,1-3-.2A47.84,47.84,0,0,1,25.25,81.14a45.26,45.26,0,0,0,18.09,3.7C68,84.84,86.69,66,86.69,42.5S68,0,43.34,0Z"/><path class="cls-1" d="M17.58,76.82a42.4,42.4,0,0,0,6.23,3.66A47.8,47.8,0,0,0,38.16,64.09a20.52,20.52,0,0,1-8.7-4.33A47.59,47.59,0,0,1,17.58,76.82Z"/><path class="cls-1" d="M39.27,20.53A48,48,0,0,0,25.93,3.43,42.92,42.92,0,0,0,19.15,7,48,48,0,0,1,30.2,24.54,20.62,20.62,0,0,1,39.27,20.53Z"/><path class="cls-1" d="M28.36,26.17A48.14,48.14,0,0,0,17.71,8a42.47,42.47,0,0,0-6.31,5.5A48.58,48.58,0,0,1,20,41.21a48.22,48.22,0,0,1-9.59,29.1,41.55,41.55,0,0,0,5.88,5.52A47.77,47.77,0,0,0,27.73,58.1a22.65,22.65,0,0,1-5.89-15.6A22.84,22.84,0,0,1,28.36,26.17Z"/><path class="cls-1" d="M6.85,41.21A49,49,0,0,1,3.42,59.3a41.29,41.29,0,0,0,5.67,9.48,48.32,48.32,0,0,0,8.49-27.57A48.66,48.66,0,0,0,10,15,41.85,41.85,0,0,0,4,24.48,49.26,49.26,0,0,1,6.85,41.21Z"/><path class="cls-1" d="M2.19,56.06A49.08,49.08,0,0,0,4.47,41.21,49.48,49.48,0,0,0,2.6,27.73a43.17,43.17,0,0,0-.41,28.33Z"/></g></g></svg>
```

Comment:
Please make the following changes:
1. Format the file
2. Remove class and style constructs as those prevent logos from being deployed on Azure.